Makefile

```diff
@@ -62,7 +62,7 @@ docker:
 	docker build -t llama-cpp-python:latest -f docker/simple/Dockerfile .
 
 run-server:
-	uvicorn --factory llama.server:app --host ${HOST} --port ${PORT}
+	python llama_cpp/server --model ${MODEL}
```

Comment:
Why this change? @domdomegg

Reply:
I don't believe the current instructions work, and this is how I got it working.

Happy to split this out to a separate PR, or for challenge that the uvicorn command does work.

Reply:
Err yes @domdomegg that's a typo, it should be `uvicorn --factory llama.server.app:create_app --host ${HOST} --port ${PORT}`, I'll merge this and fix the Makefile after.